.github/workflows/build.yml

```diff
@@ -0,0 +1,99 @@
+name: IntuneAppBuilder Deployment
+
+# TODO: Remove these comments once reviewed
+# on:
+#   schedules:
+#   - cron: "30 7 * * *"
+#     displayName: Daily job
+#     branches:
+#       include:
+#       - master
+#     always: true
+#   push:
+#     branches:
+#       - master
+#   pull_request:
+#     types: [opened, synchronize, reopened, closed]
+#     branches:
+#       - master
+
+on: workflow_dispatch
+
+env:
+  BuildConfiguration: 'Release'
+  Verbosity: 'normal'
+  ArtifactFile: 'output.nupkg'
+
+defaults:
+  runs-on: windows-latest
+  run:
+    shell: pwsh
+
+jobs:
+  Build:
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.301
+
+      - name: dotnet restore
+        run: dotnet restore --configuration $(Env:BuildConfiguration) --verbosity $(Env:Verbosity)
```

Comment:
unless the "run" step uses PowerShell by default the $ expressions won't work
also, you don't need the extra `$(` and `$env:` is usually lowercased

Reply:
I set the default shell to PowerShell at the beginning of the file.

Reply:
one thing to beware of is how PowerShell handles exit codes of external applications it launches...make sure that if the command fails the build actually fails....